packages/graphback-runtime/src/service/CRUDServiceCreator.ts

```diff
@@ -0,0 +1,34 @@
+import { PubSubEngine } from 'graphql-subscriptions';
+import { GraphbackDataProvider } from '../data/GraphbackDataProvider';
+import { PubSubConfig } from '../GraphbackPubSubModel';
+import { GraphbackMessageLogger } from '../utils/Logger';
+import { ServiceCreator } from './ServiceCreator';
+import { CRUDServiceOptions, CRUDService } from './CRUDService';
+
+export interface CRUDServiceCreatorConfig {
+  pubSub: PubSubEngine
+  logger?: GraphbackMessageLogger
+}
+/**
+ * Creator class to create a new CRUDService
+ */
+export class CRUDServiceCreator extends ServiceCreator {
```

Comment:
Any reason why this needs to be class vs function?

Reply:
+1